tariff/pun.go

```diff
@@ -123,48 +127,67 @@ func (t *Pun) Type() api.TariffType {
 }
 
 func (t *Pun) getData(day time.Time) (api.Rates, error) {
-	// Cookie Jar zur Speicherung von Cookies zwischen den Requests
+	// Cookie Jar to store cookies between requests
 	client := request.NewClient(t.log)
 	client.Jar, _ = cookiejar.New(nil)
 
-	// Erster Request
-	uri := "https://storico.mercatoelettrico.org/It/WebServerDataStore/MGP_Prezzi/" + day.Format("20060102") + "MGPPrezzi.xml"
-	resp, err := client.Get(uri)
+	// Request the ZIP file
+	uri := "https://gme.mercatoelettrico.org/DesktopModules/GmeDownload/API/ExcelDownload/downloadzipfile?DataInizio=" + day.Format("20060102") + "&DataFine=" + day.Format("20060102") + "&Date=" + day.Format("20060102") + "&Mercato=MGP&Settore=Prezzi&FiltroDate=InizioFine"
+	req, err := http.NewRequest("GET", uri, nil)
 	if err != nil {
 		return nil, err
 	}
+
+	req.Header = http.Header{
```

Comment:
Are these headers actually needed or is client.Get sufficient?

Reply:
no, the headers are needed